website/docs/developer/research/research-deep-glacier-migration.md

```diff
@@ -0,0 +1,154 @@
+---
+id: research-deep-glacier-migration
+title: Deep archive storage migration Research Notes
+description: Research Notes on deep archive migration.
+---
+
+## Overview
+
+Deep archive S3 storage type is the cheapest glacier storage type currently present in AWS S3. For archive data that does not require immediate access such as backup or disaster recovery use cases, choosing `Deep Archive Access` tier could be a wise choice.
+- Storage cost is ~$1/TB data per month. 
+- Objects in the Deep Archive Access tier are moved to the Frequent Access tier within 12 hours. Once the object is in the Frequent Access tier, users can send a GET request to retrieve the object.
+
+
+### Assumptions
+
+- List of objects already exists in orca catalog and S3 bucket.
+- Users should be able to provide some information(filename) to retrieve objects possibly at the collection level.
+
+### Initial Implementation Idea
+
+Migration should be a 2 step process- first retrieve the objects and then copy the objects to deep archive.
+
+- Retrieve all objects currently stored with Glacier Flexible Retrieval type using `Bulk Retrievel` which is the cheapest retrieval option. Bulk retrievals are typically completed within 5–12 hours. The prices are lowest among all retrieval rates - $0.0025 per GB plus $0.025 per 1,000 requests.
```

Comment:
I think bulk retrievals might be free. See ["you can request free bulk retrievals"](https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-class-intro.html)
The [pricing guide](https://aws.amazon.com/s3/pricing/) lists the cost as "n/a" which I think in this context means 0.
The 0.0025 is from the cheapest retrieval option in Deep Archive.

Reply:
from what I found, there is a fee for bulk retrieval which is the cheapest 
https://www.arqbackup.com/aws-glacier-pricing.html
https://www.msp360.com/resources/blog/amazon-glacier-pricing-explained/

but in the aws website, it shows n/a. Not sure which one to use.

Reply:
.025 per-request (as in the first link) is definitely correct. I should have clarified that I was just referring to the per GB cost. My guess is that the .0025 per GB in the msp link was either copied in incorrectly, or has become outdated since 2017.

Reply:
I concur with @AndrewDorn

Reply:
removed /GB